main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
```

Comment:
Why the `window` in this module?

How different is `window.document` from `document`?